pkgs/top-level/all-packages.nix

```diff
@@ -25994,7 +26000,10 @@ with pkgs;
 
   # Clasp Common Lisp
   clasp-common-lisp = wrapLisp {
-    pkg = callPackage ../development/compilers/clasp { };
+    pkg = callPackage ../development/compilers/clasp {
+      stdenv = clang15Stdenv;
+      inherit (llvmPackages_15) llvm libclang;
```

Comment:
[ofborg-eval](https://gist.github.com/GrahamcOfBorg/cb6cf2a15c58d427a368e1eed4c44f58) didn't like this. Hmm. I wonder why it works locally with `nix-build -A clasp-common-lisp`?